src/GeneratorState.hpp

```diff
@@ -0,0 +1,42 @@
+#ifndef GENSET_WHISPERPOWER_DDC_GENERATOR_STATE_HPP
+#define GENSET_WHISPERPOWER_DDC_GENERATOR_STATE_HPP
+
+#include <base/Time.hpp>
+#include <genset_whisperpower_ddc/GeneratorStatus.hpp>
+
+namespace genset_whisperpower_ddc {
+    struct GeneratorState {
+        base::Time time;
+
+        int rpm;
```

Comment:
`rpm` is a unit. Should be `float rotation_speed` and in rad/s (that's the standard unit for rotation speeds in Rock)